03/ListStyles.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const ListStyles = () => {
```

Comment:
To nie jest komponent więc piszemy z małej litery :P
Nie trzeba też importować `React` bo to nie komponent (nie zwraca kodu JSX) tylko normalna funkcja; P